### PR TITLE
Created Grid Component

### PR DESCRIPTION
**Pull request by anushasunil**
image

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
Looks good to me for merge and Code is also easy to understand thanks to your application of grid-areas. Great demo Anusha!!

File: Stylesheets/Components/ratings.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    border-radius: var(--container-border);
+}
+
+.rating-star{
+    margin:0;
+    position:relative;
+}
+
+.rating-container.badge {
+    position: static;
```
Comment (aishwarya-bm) [inline Stylesheets/Components/ratings.css:16]: Default position is static, is this required?

Comment (anushasunil) [reply Stylesheets/Components/ratings.css:16]: actually I am using the badge component class and this class is having position as absolute in badge.css so to change only this one property I am overriding the position property

Comment (aishwarya-bm) [reply Stylesheets/Components/ratings.css:16]: Good to know.
Code looks very clean & great view for grids!

**Review by aishwarya-bm — APPROVED** [review]

**Pull request merged**